src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
you can make destructuring from a person
```suggestion
export const Person = ({ person }) => {
  const { ... } = person;
```